beluga_ros/tools/__init__.py

```diff
@@ -0,0 +1,17 @@
+# # Copyright 2024 Ekumen, Inc.
```

Comment:
@serraramiro1 meta: consider promoting this to a `beluga_ros` Python package. Then you can use `setup.cfg` to get your scripts installed as `console_scripts` and run with `ros2 run beluga_ros occupancy_grid_to_ndt`. Check `ament_python_install_package` documentation.

Reply:
I tried but failed, even after our sync.

After thinking a bit about it, I don't think there's a big benefit in doing this, as it involves going into a deep undocumented custom path, and most online resources point to this way of installing python scripts in cmake packages.

My vote is let's get this in as-is.

Reply:
PTAL @ https://github.com/Ekumen-OS/beluga/pull/333. You were really really close, for your consideration.

> as it involves going into a deep undocumented custom path, and most online resources point to this way of installing python scripts in cmake packages.

That's a fine line to walk when it comes to ROS. Like half the ecosystem falls in the deep undocumented custom path category :sweat_smile: